.github/workflows/versionbump.yml

```diff
@@ -40,37 +40,34 @@ jobs:
       uses: actions/setup-python@v4
       with: { python-version: "3.10" }
 
-    - name: Bump version (dev push)
-      if: github.ref == 'refs/heads/develop' && github.event_name == 'push'
+    - name: Bump version
+      env:
+        GH_TOKEN: ${{ github.token }}
       run: |
         git config --local user.name "github-actions[bot]"
         git config --local user.email "github-actions[bot]@users.noreply.github.com"
-        git log -1 --pretty=%B > prevMsg.txt
 
-        if python3 bg-ci/BumpVersion.py version.json prevMsg.txt ${{ inputs.name }} true; then
-          git add version.json
-          newVer=$(python3 bg-ci/GetVersion.py version.json ${{ inputs.name }} true)
-          newMsg="dev""${newVer}"" - ""$(cat prevMsg.txt)"
-          git commit -m "${newMsg}"
-          git pull
-          git push
+        if [ ${{ github.event_name }} == 'workflow_dispatch' ]; then
```

Comment:
Future Ben and Future Meysam would appreciate a comment what this branch is distinguishing.